### PR TITLE
Update docs for buckets.

### PR DESCRIPTION
**Pull request by mjkw31**

### CODE REVIEW

**Review by vviyer — COMMENTED** [review]
inspected visually and passed

File: docs/getting_started/tre_organisation.md [hunk 1]
```diff
@@ -6,8 +6,7 @@ This [reference page](../explainers/file_structure.md) goes through the other fo
 # Your Home Folder
 Available at `/home/ivm` in your sandbox, this is your personal folder. You can use this to store any files you want to keep, but it is not backed up. If you delete a file from here, it is gone forever.
 
-If you have access to multiple sandboxes, your 
-`/home` folder will be accessible from all of them.
+<!-- If you have access to multiple sandboxes, your `/home` folder will be accessible from all of them. -->
```
Comment (donnybeelo) [inline docs/getting_started/tre_organisation.md:9]: Is this actually true then? I had no idea

**Review by vviyer — APPROVED** [review]

File: docs/getting_started/tre_organisation.md [hunk 2]
```diff
@@ -18,17 +17,37 @@ You can use your home folder for any files you want to keep, but it is not backe
...
 
 `library-red` is a google storage bucket gs://qmul-sandbox-production-library-red/ (read+write only for admins)
 
 `library-red` is where curated and raw data is stored. This is where you will find the data you need to run your analysis. There are a number of subfolders in `library-red` that contain different types of data, and each folder should be used for a specific purpose. If you find a folder that does not have a readme file, please contact the Genes and Health team to get more information on what the folder is for.
 
 ## red
-`red` is used directly by the virtual machine, and is specific to each sandbox. Users in the same sandbox can all see the contents of the red folder. Most organisations use this folder to store and run their analysis. 
+`red` is used directly by the virtual machine, and is specific to each sandbox. Users in the same sandbox can all see the contents of the red folder. Most organisations use this folder to store their analyses.
+
+`red` has slower storage than `/home/ivm` but is backed up, and we strongly recommend that you use this folder to store your data and any analysis files you are running frequently to avoid losing your work.
```
Comment (donnybeelo) [inline docs/getting_started/tre_organisation.md:31]: Make it clear that data that won't be used during some job execution should go in red, and anything they're actively working on should go in /home/ivm

File: docs/getting_started/tre_organisation.md [hunk 3]
```diff
@@ -18,17 +17,37 @@ You can use your home folder for any files you want to keep, but it is not backe
...
+
+```
+gsutil cp -r -n my_file "gs://$BUCKET_SANDBOX_IVM/"
```
Comment (donnybeelo) [inline docs/getting_started/tre_organisation.md:46]: Unintended side effect is that if someone is already in sandbox 1, they might just copy the code and find it doesnt work. Set default value to say 1.

File: docs/getting_started/tre_organisation.md [hunk 4]
```diff
@@ -18,17 +17,37 @@ You can use your home folder for any files you want to keep, but it is not backe
...
...00;border-radius:.1rem"><select style="display: block" onchange="this.nextElementSibling.firstChild.children[2].innerText='gsutil cp -r -n my_file gs://qmul-production-sandbox-'+this.value+'-red/'"><option value="1">Sandbox 1 - QMUL+WSI Core Team Desktop</option><option value="2">Sandbox 2 - External Academic Desktop</option><option value="3">Sandbox 3 - GSK Desktop</option><option value="4">Sandbox 4 - BMS Desktop</option><option value="5">Sandbox 5 - MSD Desktop</option><option value="6">Sandbox 6 - Takeda Desktop</option><option value="7">Sandbox 7 - Pfizer Desktop</option><option value="8">Sandbox 8 - S00050_FFAIR-PRS Desktop</option><option value="9">Sandbox 9 - Maze Therapeutics Desktop</option><option value="10">Sandbox 10 - Novo Nordisk Desktop</option><option value="11">Sandbox 11 - University of Exeter</option><option value="12">Sandbox 12 - Genomics PLC</option><option value="13">Sandbox 13 - AstraZeneca</option><option value="14">Sandbox 14 - External Academic, Consortium access</option><option value="15">Sandbox 15 - 5 Prime Sciences</option><option value="16">Sandbox 16 - Sandbox 16</option><option value="17">Sandbox 17 - Academic, NHS Digital access</option></select>
```
Comment (donnybeelo) [inline docs/getting_started/tre_organisation.md:43]: Ok, this is somewhere I think the widget doesn't need to exist, if that environment variable works in both old and new TRE

File: docs/getting_started/tre_organisation.md (hunk 1 above)
Comment (saeedbidi) [reply docs/getting_started/tre_organisation.md:9]: I don't think that's true. I removed it.

File: docs/getting_started/tre_organisation.md (hunk 2 above)
Comment (saeedbidi) [reply docs/getting_started/tre_organisation.md:31]: added.

**Pull request merged**